common/djangoapps/track/views/__init__.py

```diff
@@ -78,6 +79,7 @@ def user_track(request):
 
     name = _get_request_value(request, 'event_type')
     data = _get_request_value(request, 'event', {})
```

Comment:
Just following up here from a slack conversation - the frontend implementation of the `sendTrackingLogEvent` can only add arbitrary data into the `event` request value here. 

Relevant code here: https://github.com/edx/frontend-platform/blob/master/src/analytics/SegmentAnalyticsService.js#L132

I would hesitate to provide a second mechanism for passing _other_ data to be a sibling of `event` when we already have that precedent.  I'd expect that means refactoring line 82 below to get `course_id_string` out of `event` instead of as its own request value (i.e., `'courserun_key'`).

Reply:
am currently implementing that refactor